cmis/exceptions.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
```

Comment:
those above...
CMISError -> UserError
~~CMISConnectionError~~ -> don't catch the native exception